src/components/useRequest.ts

```diff
@@ -31,26 +31,27 @@ import { useUrl } from './useUrl';
   const { baseUrl } = useUrl();
   const { owner, repo } = useProjectEntity(entity);
   const { value, loading, error } = useAsync(async (): Promise<any> => {
-  const token = await auth.getAccessToken(['repo']);
-  const octokit = new Octokit({auth: token});
+    const token = await auth.getAccessToken(['repo']);
+    const octokit = new Octokit({auth: token});
 
-  const response = await octokit.request(`GET /repos/{owner}/{repo}/${requestName}`, {
-    baseUrl,
-    owner,
-    repo,
-    ...(perPage && {per_page: perPage}),
-  });
+    const response = await octokit.request(`GET /repos/{owner}/{repo}/${requestName}`, {
+      baseUrl,
+      owner,
+      repo,
+      ...(perPage && {per_page: perPage}),
+    });
 
-  const data = response.data;
+    const data = response.data;
 
-  if(showTotal) {
-    return {
-      data,
-      total: Object.values(data as number).reduce((a, b) => a + b),
+    if(showTotal) {
+      if(Object.values(data).length === 0) return null;
+      return {
+        data,
+        total: Object.values(data as number).reduce((a, b) => a + b),
```

Comment:
I'm pretty sure that the `data` variable isn't of type number, because Object.values(someNumber) should then return an empty array (number type can't have any assigned properties)

Maybe there should be a cast to `Record<string, number>` or `[key: string]: number`